kvmagent/kvmagent/test/test_parsing_multipath.py

```
@@ -1,66 +1,81 @@
-    assert not multipath.write_multipath_conf('multipath.conf')
+    assert write_multipath_conf('multipath.conf', blacklist)
+    assert not write_multipath_conf('multipath.conf')
     with open("multipath.conf",
          "r") as fd:
         text1 = multipath.parse_multipath_conf(fd)
     with open("multipath.conf.out", "r") as fd1:
         text2 = multipath.parse_multipath_conf(fd1)
-    assert cmp(text1, text2) == 0
+    assert text1 == text2
 
     #In multipath1.conf, we set the features "1 Queue" of the device_ if_ no_ path"， no_ path_ Retry 60,
```

Ruff (0.13.3)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)